bots/pr/src/main/java/org/openjdk/skara/bots/pr/CommandExtractor.java

```diff
@@ -62,6 +62,8 @@ private static String formatId(String baseId, int subId) {
             Map.entry("clean", new CleanCommand()),
             Map.entry("open", new OpenCommand()),
             Map.entry("backport", new BackportCommand()),
+            Map.entry("approval", new ApprovalCommand()),
+            Map.entry("request-approval", new RequestApprovalCommand()),
```

Comment:
... and just to be clear: `/request-approval` is an okay command, since "request" is a verb.